pootle/static/js/shared/components/AutoIcon.js

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (C) Zing contributors.
+ *
+ * This file is a part of the Zing project. It is distributed under the GPL3
+ * or later license. See the LICENSE file for a copy of the license and the
+ * AUTHORS file for copyright and authorship information.
+ */
+
+import React from 'react';
+
+import crc16 from '../utils/crc16'; // use relative path, otherwise tests will fail
+
+export function calcAbbreviation(title, length) {
```

Comment:
I understand the reason to export this function (and the other one below) is for tests, although it is not necessary at all as we use the `rewire` plugin in tests, so they can retrieve private module dependencies. See https://github.com/evernote/zing/commit/da3ba2a for an example on how to use it.